### DISCUSSION TITLE
Web GUI unavailable from different subdomain

### DISCUSSION THREAD

**Discussion by jdpadro** [Security-Onion-Solutions/securityonion#4523] [Unsupported Versions]
One of our overseas remote location has mirgrated from 16.04.6.1 to 2.3.52. There TLD is a different subdomain then ours. My jumpbox IP address was added during the installation process. I am able to initiate a web gui connection to that device using https://abc.madrid.xyz.com, since the generated cert is self-signed I am provided the obligatory "Not Secured" dialog during the cert exchange. Once this completes the connection then fails since it returns https://abc and since this device is not part of my subdomain I get a blank white screen, is there a way of adding abc.madrid.xyz.com to the nginx config so it can return the proper path allowing me to connect to the Web GUI for this remote site? 

Thanks

**Comment by jdpadro** [comment]
Issue was self resolved. Modified 
/etc/hosts file 
/opt/so/saltstack/local/global.sls  entry "url" from hostname to FQDN

I am now able to connect to the remote Web GUI

**Reply by weslambert** [reply]
Yes, `url_base` will need to be used/modified if you are trying to access the web interface by anything other than the hostname or internal IP address of the machine.
